documentation/docs/index.md

```diff
@@ -81,6 +81,33 @@ Files[IO]
   .drain
 ```
 
+### Dealing with erroneous files
+
+The default behaviour when parsing CSV files, is to terminate the stream whenever the columns of a row do not match 
+with the columns of the header. If you're dealing with CSV files that could contain these kind of errors, you can make 
+use of the `lenient` package. You will get back a `Stream` of results, where each parsed row is represented by an 
+`Either[Throwable, A]`.
+
+```scala mdoc:silent
+val content =
+  """name,age,description
+    |John Doe,47,description 1
+    |Jane Doe,50
+    |Bob Smith,80,description 2
+    |Alice Grey,78
+    |""".stripMargin
+
+Stream
+      .emit(content)
+      .covary[Fallible]
+      .through(lenient.attemptDecodeGivenHeaders[TestData]())
+      .compile
+      .toList
+
+// returns Either[Throwable, List[Either[Throwable, TestData]]]
```

Comment:
This comment should not be here, I'd like to see the mdoc output (hence removing the `silent`) in the documentation.

Reply:
I just realized this documentation belongs to the CSV documentation, not the root one. Can you please move it to [this file](https://github.com/gnieh/fs2-data/blob/main/documentation/docs/csv/index.md)?

Reply:
Good catch! I've moved it to the appropriate location and added a bit more content.

Reply:
Perfect, thanks. I'm fine if @ybasket is.